### DISCUSSION TITLE
Showing {   "detail": "Not authenticated" } when authorized

### DISCUSSION THREAD

**Discussion by TomEverson** [fastapi/fastapi#6287] [Questions]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

I have followed the documentation and still showing like that !! 

authentication.py

```python

from fastapi import APIRouter, Depends
import schemas, database , models
from sqlalchemy.orm import Session
from passlib.context import CryptContext
import jwt_token
from fastapi.security import OAuth2PasswordRequestForm

router = APIRouter(
    tags=["Authentication"]
)

get_db = database.get_db

pwd_cxt = CryptContext(schemes=["bcrypt"], deprecated="auto")

@router.post("/login")
def login(request:OAuth2PasswordRequestForm = Depends(), db: Session = Depends(get_db)):
    user = db.query(models.Users).filter(models.Users.email == request.username).first()
    if not user:
        return "User not found"
    if not pwd_cxt.verify(request.password,user.password):
        return "Incorrect Password"
    access_token = jwt_token.create_access_token(data={"sub": user.email})
    return {"access_token": access_token, "token_type": "bearer"}
```

Oauth2.py
```python
from fastapi import Depends, HTTPException, status
from fastapi.security import OAuth2PasswordBearer
import jwt_token

oauth2_scheme = OAuth2PasswordBearer(tokenUrl="login")

def get_current_user(token: str = Depends(oauth2_scheme)):
    credentials_exception = HTTPException(
        status_code=status.HTTP_401_UNAUTHORIZED,
        detail="Could not validate credentials",
        headers={"WWW-Authenticate": "Bearer"},
    )
    return jwt_token.verify_token(token,credentials_exception)
```

jwt_token.py
```python
from datetime import datetime, timedelta
from fastapi import Depends
from jose import JWTError, jwt
import schemas

SECRET_KEY = "e3926957a0582e9a4a3d17034c157008"
ALGORITHM = "HS256"
ACCESS_TOKEN_EXPIRE_MINUTES = 1440

def create_access_token(data: dict):
    to_encode = data.copy()
    expire = datetime.utcnow() + timedelta(minutes=ACCESS_TOKEN_EXPIRE_MINUTES)
    to_encode.update({"exp": expire})
    encoded_jwt = jwt.encode(to_encode, SECRET_KEY, algorithm=ALGORITHM)
    return encoded_jwt

def verify_token(token:str,credentials_exception):
    try:
        payload = jwt.decode(token, SECRET_KEY, algorithms=[ALGORITHM])
        email: str = payload.get("sub")
        if email is None:
            raise credentials_exception
        token_data = schemas.TokenData(email=email)
    except JWTError:
        raise credentials_exception
```

I have followed the documentation and still showing like that !! 

### Description

![image](https://user-images.githubusercontent.com/96673241/185754937-ebbeb1db-2119-45b8-b9ca-ffb9c77bddf7.png)
![image](https://user-images.githubusercontent.com/96673241/185754948-6473a7c3-1248-410b-a0d7-0a18b5160353.png)

### Operating System

Windows

### FastAPI Version

0.79.0

### Python Version

3.10

**Comment by JarroVGIT** [comment]
Please reformat your issue properly so others can see the images etc.

**Comment by iudeen** [comment]
I don't see `get_current_user` being called anywhere. That might be an issue?
